source/rust_verify/src/rust_to_vir_base.rs

```diff
@@ -857,56 +860,17 @@ pub(crate) fn param_ty_to_vir_name(param: &rustc_middle::ty::ParamTy) -> String
     }
 }
 
-pub(crate) fn check_generics_bounds<'tcx>(
+pub(crate) fn process_predicate_bounds<'tcx, 'a>(
```

Comment:
Note that the diff algorithm considered this a rename, where it's in fact factoring out common code and `check_generic_bounds` is otherwise unchanged (you can see it being re-added later).